### PR TITLE
[Workers] playground revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/workers/src/content/learning/playground.md [hunk 1]
```diff
@@ -25,45 +25,45 @@ async function handleRequest(request) {
...
-To get familiar with Workers, we recommend playing around in the Playground, borrowing bits from [Examples](/examples) section of the documentation. This will give you a taste of what Workers can do. It’s pretty [amazing](https://www.cloudflare.com/learning/serverless/why-use-serverless/).
+To get familiar with Workers, experiment with the Playground by borrowing [Examples](/examples) from the documentation. This will allow you to experience firsthand [what Workers can do](https://www.cloudflare.com/learning/serverless/why-use-serverless/).
 
 --------------------------------
 
-## Using the playground
+## Using the Playground
 
-When you have code you are ready to test, click the button to preview at the bottom of the script panel.
+When you have code you are ready to test, select **Update** to preview at the bottom of the script panel.
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/playground.md]:
```suggestion
When you have code you are ready to test, select **Save and Deploy** to preview at the bottom of the script panel.
```

File: products/workers/src/content/learning/playground.md [hunk 2]
```diff
@@ -25,45 +25,45 @@ async function handleRequest(request) {
...
 
-Now you should be able to see a preview on the right side of _that exact code_ running just as it would in a browser. Enter your website’s address in the right section to preview the Workers script running _on that site_.
+Now you should be able to see a preview on the right side of that exact code running just as it would in a browser. Enter your website’s address in the right section to preview the Workers script running on that site.
 
 You can modify the script and click the preview button to view the effect on the request.
 
-To test a raw HTTP request — not in an HTML previewer, for example to test a `POST` request — go to the **Test** tab in the previewer. To run the HTTP preview, we will need to click update preview and run test.
+To test a raw HTTP request — not in an HTML previewer, for example to test a `POST` request — go to the **Test** tab in the previewer. To run the HTTP preview, select **Update**.
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/playground.md]:
```suggestion
To test a raw HTTP request — not in an HTML previewer, for example, to test a `POST` request — go to the **HTTP** in the previewer. To run the HTTP preview, select **Send/Run Test**.
```

File: products/workers/src/content/learning/playground.md [hunk 3]
```diff
@@ -25,45 +25,47 @@ async function handleRequest(request) {
...
+This is the least complex Worker you can write. When the Worker receives a request, the `fetch` event is dispatched. [RespondWith](/learning/fetch-event-lifecycle#respondwith) intercepts the event, promising to return the result of the `handleRequest` function to the client. Finally, `handleRequest` is actually called, and it returns a text [response](/runtime-apis/response) of `"Hello world"` which is delivered back to the client.
 
-Check out the reference for [addEventListener](/runtime-apis/add-event-listener), [FetchEvent](/runtime-apis/fetch-event) and [FetchEvent lifecycle](/learning/fetch-event-lifecycle) to learn more.
+Refer to the reference for [addEventListener](/runtime-apis/add-event-listener), [FetchEvent](/runtime-apis/fetch-event) and [FetchEvent lifecycle](/learning/fetch-event-lifecycle) to learn more.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/playground.md]:
```suggestion
Refer to the reference for [addEventListener](/runtime-apis/add-event-listener), [FetchEvent](/runtime-apis/fetch-event), and [FetchEvent lifecycle](/learning/fetch-event-lifecycle) to learn more.
```

File: products/workers/src/content/learning/playground.md [hunk 4]
```diff
@@ -25,45 +25,47 @@ async function handleRequest(request) {
...
-To get familiar with Workers, we recommend playing around in the Playground, borrowing bits from [Examples](/examples) section of the documentation. This will give you a taste of what Workers can do. It’s pretty [amazing](https://www.cloudflare.com/learning/serverless/why-use-serverless/).
+To get familiar with Workers, experiment with the Playground by borrowing [Examples](/examples) from the documentation. This will allow you to experience firsthand [what Workers can do](https://www.cloudflare.com/learning/serverless/why-use-serverless/).
 
 --------------------------------
 
-## Using the playground
+## Using the Playground
 
-When you have code you are ready to test, click the button to preview at the bottom of the script panel.
+To access the dashboard Playground, log into your Cloudflare account and go to **Account Home** > **Workers** > **your Worker script** > **Quick edit**.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/playground.md:44]: Not sure if it's "dashboard Playground" or "Playground dashboard" instead (I don't know the UI).

File: products/workers/src/content/learning/playground.md [hunk 5]
```diff
@@ -25,45 +25,47 @@ async function handleRequest(request) {
...
 
-To test a raw HTTP request — not in an HTML previewer, for example to test a `POST` request — go to the **Test** tab in the previewer. To run the HTTP preview, we will need to click update preview and run test.
+To test a raw HTTP request — not in an HTML previewer, for example, to test a `POST` request — go to **HTTP**. To run the HTTP preview, select **Run Test**.
 
 --------------------------------
 
 ## Devtools
 
-For debugging Workers inside the playground, use out the developer tools at the bottom of the preview panel. The developer tools for Workers previewer works similar to the developer tools in Chrome or Firefox.
+For debugging Workers inside the playground, use the the developer tools at the bottom of the Playground's preview panel. The developer tools for the Workers Playground works similar to the developer tools in Chrome or Firefox.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/playground.md]: Should the first occurrence of "playground" also be "Playground"?

File: products/workers/src/content/learning/playground.md [hunk 6]
```diff
@@ -25,45 +25,47 @@ async function handleRequest(request) {
...
+**Network** shows the outgoing requests from your Workers script — that is, any calls to `fetch` inside your script.
 
 ### Console Logs
 
 The console displays the output of any calls to `console.log` that were called for the current preview run as well as any other preview runs in that session.
 
 ### Sources
 
-The sources that make up your Workers script. Note that access KV, text, and secret bindings are only accessible when authenticated with an account. This means you must be logged in to the dashboard, or use `wrangler dev` with your account credentials.
+**Sources** is the sources that make up your Workers script. Note that access KV, text, and secret bindings are only accessible when authenticated with an account. This means you must be logged in to the dashboard, or use `wrangler dev` with your account credentials.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/playground.md]:
```suggestion
**Sources** displays the sources that make up your Workers script. Note that access KV, text, and secret bindings are only accessible when authenticated with an account. This means you must be logged in to the dashboard, or use `wrangler dev` with your account credentials.
```
"Sources is the sources" sounds very awkward... Tried to come up with an alternative.

File: products/workers/src/content/learning/playground.md [hunk 7]
```diff
@@ -25,45 +25,47 @@ async function handleRequest(request) {
...
 
 <!--
```
Comment (pedrosousa) [inline products/workers/src/content/learning/playground.md:72]: Are you keeping this commented content for now?

**Review by pedrosousa — APPROVED** [review]

File: products/workers/src/content/learning/playground.md (hunk 5 above)
Comment (deadlypants1973) [inline products/workers/src/content/learning/playground.md]:
```suggestion
For debugging Workers inside the Playground, use the developer tools at the bottom of the Playground's preview panel. The developer tools for the Workers Playground works similar to the developer tools in Chrome or Firefox.
```

File: products/workers/src/content/learning/playground.md [hunk 8]
```diff
@@ -25,45 +25,47 @@ async function handleRequest(request) {
...
-To get familiar with Workers, we recommend playing around in the Playground, borrowing bits from [Examples](/examples) section of the documentation. This will give you a taste of what Workers can do. It’s pretty [amazing](https://www.cloudflare.com/learning/serverless/why-use-serverless/).
+To get familiar with Workers, experiment with the Playground by borrowing [Examples](/examples) from the documentation. This will allow you to experience firsthand [what Workers can do](https://www.cloudflare.com/learning/serverless/why-use-serverless/).
 
 --------------------------------
 
-## Using the playground
+## Using the Playground
 
-When you have code you are ready to test, click the button to preview at the bottom of the script panel.
+To access the dashboard Playground, log into your Cloudflare account and go to **Account Home** > **Workers** > **your Worker script** > **Quick edit**.
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/playground.md:44]:
```suggestion
There are two versions of the Playground available. The [browser Playground](cloudflareworkers.com) and the dashboard Playground, also known as the previewer. 

To access the dashboard Playground, log into your Cloudflare account and go to **Account Home** > **Workers** > **your Worker script** > **Quick edit**.
```

**Pull request merged**